amplify/backend/api/householdapp/schema.graphql

```diff
@@ -222,8 +223,8 @@ type Mutation {
   createNewHouseHold(houseHoldName: String!): String @function(name: "createNewHouseHold-${env}") @auth(rules: [{ allow: private }])
   addUserToHouseHold(cognitoUsername: String!, houseHoldId: String!): AddUserToHouseHoldResponse @function(name: "addUserToHouseHold-${env}") @auth(rules: [{ allow: custom }])
   removeUserFromHouseHold(houseHoldId: String!, houseHoldMemberId: String!): String @function(name: "removeUserFromHouseHold-${env}") @auth(rules: [{ allow: custom }])
-  createEventHandler(calendarId: String!, taskId: String, frequency: FREQUENCY!, sourceDate: AWSDateTime!, endDate: AWSDateTime!, eventType: EVENTTYPE!): String @function(name: "createEventHandler-${env}") @auth(rules: [{ allow: custom }])
-  updateEventHandler(eventHandlerId: String!, calendarId: String!, taskId: String, frequency: FREQUENCY!, sourceDate: AWSDateTime!, endDate: AWSDateTime!, eventType: EVENTTYPE!): String @function(name: "updateEventHandler-${env}") @auth(rules: [{ allow: custom }])
+  createEventHandler(calendarId: String!, taskId: String, frequency: FREQUENCY!, sourceDate: AWSDateTime!, endDate: AWSDateTime!, eventType: EVENTTYPE!, title: String!): String @function(name: "createEventHandler-${env}") @auth(rules: [{ allow: custom }])
+  updateEventHandler(eventHandlerId: String!, calendarId: String!, taskId: String, frequency: FREQUENCY!, sourceDate: AWSDateTime!, endDate: AWSDateTime!, eventType: EVENTTYPE!, title: String!): String @function(name: "updateEventHandler-${env}") @auth(rules: [{ allow: custom }])
```

Comment:
Do we also want to add completeEvent to the schema? Or is it only going to be invoked from within completeTask?

Reply:
Just read the other response which answered this question. Thanks.